cli/internal/util/run_opts.go

```diff
@@ -12,6 +14,11 @@ const (
 	Strict EnvMode = "Strict"
 )
 
+// MarshalText implements TextMarshaler for the struct.
+func (s EnvMode) MarshalText() (text []byte, err error) {
+	return []byte(strings.ToLower(string(s))), nil
+}
```

Comment:
Niice, I was wondering if there was a cheap way to do this. `JSON.Marshal()` works with this?

Reply:
Yup. This is how we're going to make the global hash portable to Rust.

Reply:
Reader: this was _not_ how that was going to happen.